packages/data-stores/src/reader/types/index.ts

```diff
@@ -1,9 +1,12 @@
 export type EmailFormatType = 'html' | 'text';
 
+export type DeliveryWindowDayType = 0 | 1 | 2 | 3 | 4 | 5 | 6;
+export type DeliveryWindowHourType = 0 | 2 | 4 | 6 | 8 | 10 | 12 | 14 | 16 | 18 | 20 | 22;
```

Comment:
We already have these two types defined inside the `subscription-manager` package: `packages/subscription-manager/src/components/fields/DeliveryWindowInput/DeliveryWindowInput.tsx`.

What is not clear to me is whether it is better practice to:
  A) have the types defined in both places
  B) keep the types defined in `subscription-manager` package and just import it to `data-stores`
  C) keep the types defined in `data-stores` package and just import it to `subscription-manager`
  D) some other solution

Reply:
I would say C, because I would expect the data-stores to be the source of truth. 

@mashikag what do you think?

Reply:
I would also go with C for the same reason as @TimBroddin mentioned above.

Reply:
Do you have any thoughts on the types definition above as well, @TimBroddin? Thanks! 🙂

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/74763/commits/a85ff4e5d40be2789286774f38b9ea0f4b4e0e42.